kiwi/storage/subformat/vagrant_virtualbox.py

```diff
@@ -0,0 +1,432 @@
+        CIMOSType_CIMOS_Inferno = 47,
+        CIMOSType_CIMOS_QNX = 48,
+        CIMOSType_CIMOS_EPOC    = 49,
+        CIMOSType_CIMOS_IxWorks = 50,
+        CIMOSType_CIMOS_VxWorks = 51,
+        CIMOSType_CIMOS_MiNT    = 52,
+        CIMOSType_CIMOS_BeOS    = 53,
+        CIMOSType_CIMOS_HPMPE   = 54,
+        CIMOSType_CIMOS_NextStep    = 55,
+        CIMOSType_CIMOS_PalmPilot   = 56,
+        CIMOSType_CIMOS_Rhapsody    = 57,
+        CIMOSType_CIMOS_Windows2000 = 58,
+        CIMOSType_CIMOS_Dedicated   = 59,
+        CIMOSType_CIMOS_OS390   = 60,
+        CIMOSType_CIMOS_VSE = 61,
+        CIMOSType_CIMOS_TPF = 62,
+        CIMOSType_CIMOS_WindowsMe   = 63,
+        CIMOSType_CIMOS_CalderaOpenUNIX = 64,
+        CIMOSType_CIMOS_OpenBSD = 65,
+        CIMOSType_CIMOS_NotApplicable   = 66,
+        CIMOSType_CIMOS_WindowsXP   = 67,
+        CIMOSType_CIMOS_zOS = 68,
+        CIMOSType_CIMOS_MicrosoftWindowsServer2003  = 69,
+        CIMOSType_CIMOS_MicrosoftWindowsServer2003_64   = 70,
+        CIMOSType_CIMOS_WindowsXP_64    = 71,
+        CIMOSType_CIMOS_WindowsXPEmbedded   = 72,
+        CIMOSType_CIMOS_WindowsVista    = 73,
+        CIMOSType_CIMOS_WindowsVista_64 = 74,
+        CIMOSType_CIMOS_WindowsEmbeddedforPointofService    = 75,
+        CIMOSType_CIMOS_MicrosoftWindowsServer2008  = 76,
+        CIMOSType_CIMOS_MicrosoftWindowsServer2008_64   = 77,
+        CIMOSType_CIMOS_FreeBSD_64  = 78,
+        CIMOSType_CIMOS_RedHatEnterpriseLinux   = 79,
+        CIMOSType_CIMOS_RedHatEnterpriseLinux_64    = 80,
+        CIMOSType_CIMOS_Solaris_64  = 81,
+        CIMOSType_CIMOS_SUSE    = 82,
+        CIMOSType_CIMOS_SUSE_64 = 83,
+        CIMOSType_CIMOS_SLES    = 84,
+        CIMOSType_CIMOS_SLES_64 = 85,
+        CIMOSType_CIMOS_NovellOES   = 86,
+        CIMOSType_CIMOS_NovellLinuxDesktop  = 87,
+        CIMOSType_CIMOS_SunJavaDesktopSystem    = 88,
+        CIMOSType_CIMOS_Mandriva    = 89,
+        CIMOSType_CIMOS_Mandriva_64 = 90,
+        CIMOSType_CIMOS_TurboLinux  = 91,
+        CIMOSType_CIMOS_TurboLinux_64   = 92,
+        CIMOSType_CIMOS_Ubuntu  = 93,
+        CIMOSType_CIMOS_Ubuntu_64   = 94,
+        CIMOSType_CIMOS_Debian  = 95,
+        CIMOSType_CIMOS_Debian_64   = 96,
+        CIMOSType_CIMOS_Linux_2_4_x = 97,
+        CIMOSType_CIMOS_Linux_2_4_x_64  = 98,
+        CIMOSType_CIMOS_Linux_2_6_x = 99,
+        CIMOSType_CIMOS_Linux_2_6_x_64  = 100,
+        CIMOSType_CIMOS_Linux_64    = 101,
+        CIMOSType_CIMOS_Other_64    = 102,
+        // types added with CIM 2.25.0 follow:
+        CIMOSType_CIMOS_WindowsServer2008R2 = 103,
+        CIMOSType_CIMOS_VMwareESXi = 104,
+        CIMOSType_CIMOS_Windows7 = 105,
+        CIMOSType_CIMOS_CentOS = 106,
+        CIMOSType_CIMOS_CentOS_64 = 107,
+        CIMOSType_CIMOS_OracleEnterpriseLinux = 108,
+        CIMOSType_CIMOS_OracleEnterpriseLinux_64 = 109,
+        CIMOSType_CIMOS_eComStation = 110
+        // no new types added with CIM 2.26.0
```

Comment:
This is going to really suck, because VirtualBox and friends have lists that change quite often, and extra distro patches add more entries...